diff --git org.chico.learnciat.diagram/src/lciat/diagram/edit/parts/IndividualVisualizationEditPart.java org.chico.learnciat.diagram/src/lciat/diagram/edit/parts/IndividualVisualizationEditPart.java
index 1e66957..b241750 100644
--- org.chico.learnciat.diagram/src/lciat/diagram/edit/parts/IndividualVisualizationEditPart.java
+++ org.chico.learnciat.diagram/src/lciat/diagram/edit/parts/IndividualVisualizationEditPart.java
@@ -132,1 +132,1 @@
-			locator.setBorderItemOffset(new Dimension(-20, -20));
+			locator.setBorderItemOffset(new Dimension(-8, -8));
